source/Nevermore/Querying/AST/ISelectColumns.cs

```diff
@@ -71,6 +71,23 @@ public class SelectAllSource : ISelectColumns
         public override string ToString() => GenerateSql();
     }
 
+    public class SelectAllJsonColumnLast : ISelectColumns
+    {
+        readonly string schema;
+        readonly string tableName;
+        readonly ITableColumnsCache tableColumnsCache;
+
+        public SelectAllJsonColumnLast(string schema, string tableName, ITableColumnsCache tableColumnsCache)
```

Comment:
If we know the tablename at this point the schema should be guaranteed by now. Could we pass a readonly collection of resolved columns instead of the service?